eppo_core/src/eval/eval_details_builder.rs

```diff
@@ -302,11 +304,13 @@ impl EvalAssignmentVisitor for EvalDetailsBuilder {
 }
 
 impl<'b> EvalAllocationVisitor for EvalAllocationDetailsBuilder<'b> {
-    type RuleVisitor<'a> = EvalRuleDetailsBuilder<'a>
+    type RuleVisitor<'a>
+        = EvalRuleDetailsBuilder<'a>
     where
         Self: 'a;
 
-    type SplitVisitor<'a> = EvalSplitDetailsBuilder<'a>
+    type SplitVisitor<'a>
+        = EvalSplitDetailsBuilder<'a>
```

Comment:
i just ran `cargo fmt` which seems to have modified this file automatically

Reply:
Sounds good. Not critical but it would help slightly with reviewing if formatting changes were a separate PR